packages/webdriverio/test/axe-webdriverio.spec.ts

```diff
@@ -119,7 +119,9 @@ describe('@axe-core/webdriverio', () => {
         };
 
         client = await webdriverio.remote(
-          protocol === 'webdriver' ? { ...options, port } : options
+          protocol === 'webdriver'
+            ? { ...options, port, hostname: 'localhost' }
```

Comment:
Fixes the [bug in webdriverio v8.14.1](https://github.com/webdriverio/webdriverio/issues/11998) where they changed `hostname` from `localhost` to `0.0.0.0`, which chromedriver does not like.